api/analyzedphenotypes.cv.api.inc

```diff
@@ -46,7 +46,8 @@ function analyzedphenotypes_cvprop($property = null, $parameter = null) {
     'get_cvterm' => array(
       'func'  => 'ap_get_cvterm',
       'param' => array('cvterm_id' => $parameter['cvterm_id'], 'genus' => $parameter['genus']),
-      'opts'  => array('resultset' => 'full'),
+      'opts'  => array('resultset' => 'fullset'),
```

Comment:
I think you need to change `resultset` => `dataset` here.

Reply:
missed that one :)